docs/sidebars.ts

```diff
@@ -3,83 +3,94 @@ import type {SidebarsConfig} from "@docusaurus/plugin-content-docs";
 const sidebars: SidebarsConfig = {
   tutorialSidebar: [
     "index",
-    "introduction",
-    {
-      type: "doc",
-      label: "Security",
-      id: "security",
-    },
-    {
-      type: "category",
-      label: "Getting Started",
-      items: ["getting-started/quick-start", "getting-started/installation", "getting-started/starting-a-node"],
-    },
-    "data-retention",
-    {
-      type: "category",
-      label: "Beacon Node",
-      items: [
-        "beacon-management/beacon-cli",
-        "beacon-management/networking",
-        "beacon-management/mev-and-builder-integration",
-        "beacon-management/syncing",
-      ],
-    },
     {
       type: "category",
-      label: "Validator",
+      label: "Run A Node",
```

Comment:
why is this collapsed? The content in "Run a node" is by far the most viewed, see https://github.com/ChainSafe/lodestar/issues/6550#issuecomment-2148635954. And it should be easy to access for users